content/plugins/fabric.md

```diff
@@ -229,6 +237,7 @@ node_templates:
               host_string: 192.168.10.13
               user: some_username
               key_filename: /path/to/key/file
+              sudo_prefix: 'mysudo -c'
```

Comment:
consider adding an example with `sudo_prefix` and `use_sudo` combined, as putting this one here, doesn't actually do anything